packages/typescript-estree/tests/lib/__snapshots__/semantic-diagnostics-enabled.test.ts.snap

```diff
@@ -1,5 +1,7 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
+exports[`Parse all fixtures with "errorOnTypeScriptSyntacticAndSemanticIssues" enabled fixtures/declaration/ClassDeclaration/fixtures/_error_/export-missing-name/fixture 1`] = `[Error: Cannot calculate TypeScript semantic issues without a valid project.]`;
```

Comment:
lol this test should really just be deleted - it's clearly broken and the fact we didn't catch this shows that it provides zero value for us